distributed/bokeh/scheduler.py

```diff
@@ -288,6 +288,7 @@ def __init__(self, scheduler, width=600, **kwargs):
                         x='nbytes-half', y='y',
                         width='nbytes', height=1,
                         color='nbytes-color')
+            nbytes.axis[0].ticker = BasicTicker(mantissas=[1,256,512], base=1024)
```

Comment:
I'm curious, can you explain what this is doing so that I can replicate this in the future?